core-web/libs/edit-content/src/lib/fields/dot-edit-content-fields.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+
+import { DotEditContentTextAreaComponent } from './dot-edit-content-text-area/dot-edit-content-text-area.component';
+import { DotEditContentTextFieldComponent } from './dot-edit-content-text-field/dot-edit-content-text-field.component';
+
+@NgModule({
+    declarations: [],
+    imports: [DotEditContentTextAreaComponent, DotEditContentTextFieldComponent],
+    exports: [DotEditContentTextAreaComponent, DotEditContentTextFieldComponent]
+})
+export class DotEditContentFieldsModule {}
```

Comment:
Why a Module and no Standalone?

Reply:
We will add more fields, it's easier to import them all from a module.